### PR TITLE
Consistently use ICE candidate instead of ICE host candidate

### PR DESCRIPTION
**Pull request by jeroendeb**
PR for #18

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Specifically an ICE agent using an interface with both IPv4 and IPv6 addresses M
 ICE Candidate Processing {#processing}
 ----------------------------
 
-For any remote host ICE candidate received by the ICE agent, the following procedure is used:
+For any remote ICE host candidate received by the ICE agent, the following procedure is used:
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: I think we should just drop 'host' entirely here. We won't worry about the type in this stage.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -177,11 +177,11 @@ APIs leaking IP addresses
 
 When there is no user consent, the following filtering should be done to prevent private IP address leakage:
 
-1. host ICE candidates with an IP address are not exposed as ICE candidate events.
+1. ICE host candidates with an IP address are not exposed as ICE candidate events.
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: This section is obsoleted by #22

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ The ICE agent SHOULD, if available, use the first IPv6 address resolved, otherwi
 A peer-reflexive remote candidate could be learned and constructed from the
 source transport address of the STUN Binding request as an ICE connectivity
 check. The peer-reflexive candidate could share the same address as a remote
-host ICE candidate that will be signaled or has been signaled, received and is
+ICE host candidate that will be signaled or has been signaled, received and is
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: This should probably say "remote mDNS candidate"

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 4]
```diff
@@ -138,13 +138,13 @@ Specifically, an ICE agent using an interface with both IPv4 and IPv6 addresses
...
 
-For any remote host ICE candidate received by the ICE agent, the following procedure is used:
+For any remote ICE candidate received by the ICE agent, the following procedure is used:
 
 1. If the connection-address field value of the ICE candidate does not end with ".local" or if the value contains more than one ".", then process the candidate as defined in {{RFC8445}}.
 
 2. Otherwise, resolve the candidate using mDNS.
 
-3. If it resolves to an IP address, replace the value of the ICE host candidate by the resolved IP address and continue processing of the candidate.
+3. If it resolves to an IP address, replace the value of the ICE candidate by the resolved IP address and continue processing of the candidate.
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion
3. If it resolves to an IP address, replace the mDNS hostname of the ICE candidate with the resolved IP address and continue processing of the candidate.
```

**Comment by jeroendeb** [comment]
Seems all feedback is addresses?

**Pull request merged**